### PR TITLE
{tools}[intel/2015a] IOR 2.10.3 with HDF5 support

### PR DESCRIPTION
**Pull request by jordiblasco**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/i/IOR/IOR-2.10.3-intel-2015a-hdf5.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+version = "2.10.3"
+versionsuffix= "-hdf5"
+
+homepage = 'http://sourceforge.net/projects/ior-sio/'
+description = """ The IOR software is used for benchmarking parallel file systems 
+using POSIX, MPIIO, or HDF5 interfaces. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+dependencies = [
```
Comment (boegel) [inline easybuild/easyconfigs/i/IOR/IOR-2.10.3-intel-2015a-hdf5.eb]: please put this below the`sources` block

File: easybuild/easyconfigs/i/IOR/IOR-2.10.3-intel-2015a-hdf5.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+sources = [SOURCE_TGZ]
+
+prebuildopts = 'CCFLAGS_HDF5="-DH5_USE_16_API" CC=h5pcc'
+
+# gmake posix      -- IOR with only POSIX interface
+# gmake mpiio      -- IOR with only POSIX and MPIIO interfaces
+# gmake hdf5       -- IOR with POSIX, MPIIO, and HDF5 interfaces
+# gmake ncmpi      -- IOR with POSIX, MPIIO, and NCMPI interfaces
+# gmake all        -- IOR with POSIX, MPIIO, HDF5, and NCMPI interfaces
+buildopts = ' -e hdf5 '
```
Comment (boegel) [inline easybuild/easyconfigs/i/IOR/IOR-2.10.3-intel-2015a-hdf5.eb]: why `-e`?

**Comment by migueldiascosta** [comment]
@jordiblasco, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**